Makefile

```diff
@@ -198,15 +198,27 @@ $(1)-test: musl $(GEN_RS) $(GEN_TS)
 
 .PHONY: $(1)-test-all
 $(1)-test-all: musl $(GEN_RS) $(GEN_TS)
+ifeq ($(uid), 0)
+	$(cargo) test $(2) -p $(1) -- --include-ignored
```

Comment:
should we define a `root_cargo = sudo -E $(cargo)` and use that everywhere that we need root cargo?